cfme/tests/test_rest.py

```diff
@@ -22,7 +22,7 @@
 
 
 @pytest.fixture(scope="module")
-def provider():
+def a_provider():
```

Comment:
I would not call it setup_ ... I think just a_cloud_provider[_modscope], an_infra_provider, ...